multiversion/src/main/scala/multiversion/resolvers/ContextualDependency.scala

```diff
@@ -0,0 +1,10 @@
+package multiversion.resolvers
+
+import coursier.core.Dependency
+import multiversion.configs.ModuleConfig
+
+final case class ContextualDependency(
```

Comment:
@eed3si9n Do you have a suggestion for a better name? The exclusions and dependencies of the root module this dependency comes from constitute the context in which a dependency was resolved.

Reply:
AugmentedDependecy?